manual_labeling.py

```diff
@@ -0,0 +1,108 @@
+import sys
+import os
+import argparse
+import json
+import subprocess
+
+
+def get_parser():
+    # Mandatory arguments
+    parser = argparse.ArgumentParser(
+        description=" ",
+        epilog="EXAMPLES:\n",
+        add_help=None,
+        prog=os.path.basename(__file__).strip('.py'))
+
+    mandatoryArguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatoryArguments.add_argument(
+        '-file',
+        required=True,
+        help="Json file containing list of path to image file",
+    )
+    mandatoryArguments.add_argument(
+            '-path',
+            required=True,
+            help="path to bids folder",)
+    mandatoryArguments.add_argument(
+        '-author',
+        required=True,
+        help="Author name for json file",
+    )
+    optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        '-correct',
+        default=0,
+        help=" if this is activated the -ilabel option will be used and therefore existing file will be open")
+    optional.add_argument(
+            '-o',
+            help="output path. if empty it will save in BIDS_path/label/derivatives/sub/anat ")
+    
+    return parser
+
+
+def main(args=None):
+    if args is None:
+        args = None if sys.argv[1:] else ['--help']
+    parser = get_parser()
+    arguments = parser.parse_args(args=args)
+    file_path = arguments.file
+    author_name = arguments.author
+    correct = arguments.correct
+    json_content = {"author": author_name, "label": "labels-disc-manual"}
+    list_of_subj = [line.rstrip('\n') for line in open(file_path)]
+    derivatives_base = arguments.path  # file path is BIDS: last 3 elements are /sub-xx/anat/FILENAM
+    derivatives_path = derivatives_base + 'derivatives/labels'
+    if arguments.o is not None:
+        out_path = argument.o

```

Comment:
Because:

```
(base) parker:manual-labeling alfoi_admin$ /Users/alfoi_admin/opt/miniconda3/bin/python /Users/alfoi_admin/code/manual-labeling/manual_labeling.py -file /Users/alfoi_admin/Desktop/test/log.txt -author Alexandru -path /Volumes/sct_testing/large/ -correct 1 -o /Users/alfoi_admin/Desktop/test
Traceback (most recent call last):
  File "/Users/alfoi_admin/code/manual-labeling/manual_labeling.py", line 108, in <module>
    main()
  File "/Users/alfoi_admin/code/manual-labeling/manual_labeling.py", line 56, in main
    out_path = argument.o
NameError: name 'argument' is not defined
```

And if you only fix that issue you and up with this one:
```
(base) parker:manual-labeling alfoi_admin$ /Users/alfoi_admin/opt/miniconda3/bin/python /Users/alfoi_admin/code/manual-labeling/manual_labeling.py -file /Users/alfoi_admin/Desktop/test/log.txt -author Alexandru -path /Volumes/sct_testing/large/ -correct 1 -o /Users/alfoi_admin/Desktop/test

--
Spinal Cord Toolbox (4.2.2)

Traceback (most recent call last):
  File "/Users/alfoi_admin/code/manual-labeling/manual_labeling.py", line 108, in <module>
    main()
  File "/Users/alfoi_admin/code/manual-labeling/manual_labeling.py", line 84, in main
    i=i+1
UnboundLocalError: local variable 'i' referenced before assignment
```